backend/app/Actions/Sessions/GetAvgSessionResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Sessions;
+
+use Illuminate\Support\Collection;
+
+final class GetAvgSessionResponse
+{
+    private $avgSession;
+
+    public function __construct(Collection $avgSession)
+    {
+        $this->avgSession = $avgSession;
+    }
+
+    public function avgSession(): int
+    {
+        return (int) $this->avgSession->first()->avg;
```

Comment:
DTO (Data Transfer Object) mustn't calculate anything, just retun a number. So calculation should be moved to the action